features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/InlineMessage.jsx

```diff
@@ -50,7 +50,7 @@ class InlineMessage extends React.Component {
         const { classes, type } = this.props;
         const messgeType = type || 'info';
         return (
-            <Paper className={classes.root} elevation={1} {...this.props}>
+            <Paper elevation={2} className={classes.root} elevation={1} {...this.props}>
```

Comment:
there are 2 elevation props here shall we remove one

Reply:
fixed https://github.com/wso2/carbon-apimgt/pull/7147/commits/158d1d591856515f2507b662e9cf02ae24bb9d74